lgc/state/LgcContext.cpp

```diff
@@ -153,6 +153,7 @@ void LgcContext::initialize() {
   setOptionDefault("amdgpu-unroll-max-block-to-analyze", "20");
   setOptionDefault("unroll-max-percent-threshold-boost", "1000");
   setOptionDefault("unroll-allow-partial", "1");
+  setOptionDefault("enable-phi-of-ops", "0");
```

Comment:
Consider opening a github issue that explains why this is disabled and what the blocking bugs are. Then, we could have the link to this issue in a comment, so that we don't lose track of it.

Reply:
yes, that's a good idea, see #1206

Reply:
Thanks. Can you link to this issue in a code comment?